### PR TITLE
CPDRP-317 fix missing list classes

### PR DESCRIPTION
**Pull request by tonyheadford**
### Context
[Jira](https://dfedigital.atlassian.net/browse/CPDRP-317)
Fix styles on list on sign-in page

### Changes proposed in this pull request
Add correct govuk styles to list on sign-in page 

### Guidance to review
Go to sign-in page, the list should be correctly formatted with the right styles

### Review Checks
- [ ] All pages have automated accessibility checks via cypress
- [ ] All pages have visual tests via cypress + percy
- [ ] All `School` queries are correctly scoped - `eligible` when they need to be

### How can I view this in a review app?
image

### CODE REVIEW

File: app/views/start/index.html.erb [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <p>Your school must complete these steps before your early career teachers (ECTs) start their statutory induction programme.
       For example, if your ECTs are starting in September, you need to complete these steps before then.</p>
 
-    <%= govuk_link_to "Start", new_user_session_path, button: true %>
+    <%= govuk_start_now_button text: "Start now", href: new_user_session_path %>
```
Comment (kicferk1) [inline app/views/start/index.html.erb:10]: Is that in the right PR?

**Review by kicferk1 — APPROVED** [review]
Don't get me wrong, I am happy to have two small visual fixes in one PR, but it looks like one of them has a PR for itself https://github.com/DFE-Digital/early-careers-framework/pull/564

File: app/views/start/index.html.erb (hunk 1 above)
Comment (tonyheadford) [reply app/views/start/index.html.erb:10]: Good spot! How did I manage that :(

Comment (tonyheadford) [reply app/views/start/index.html.erb]: Removed other changes

**Pull request merged**